app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,10 @@
+from app.players.player import Player
+
+
+class Dwarf(Player):
```

Comment:
![image](https://user-images.githubusercontent.com/103201010/210085989-d1e9c984-0310-47a1-974e-4ba33739c250.png)
Dwarf class should be also inherited from abc